Neos.Flow/Classes/Command/RoutingCommandController.php

```diff
@@ -100,7 +95,7 @@ public function listCommand(): void
     public function showCommand(int $index): void
     {
         /** @var Route[] $routes */
-        $routes = $this->router->getRoutes();
+        $routes = Routes::fromConfiguration($this->configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_ROUTES));
```

Comment:
we can use flow inject https://github.com/neos/flow-development-collection/blob/a77393a42c45c2456124ded78fea54d0727613a1/Neos.Flow/Classes/Annotations/InjectConfiguration.php#L59
or objects yaml pick as you fit